catalog/nsili/catalog-nsili-orb-impl/src/main/java/org/codice/alliance/nsili/orb/impl/CorbaOrbImpl.java

```diff
@@ -83,21 +107,18 @@ public void shutdown() {
         orb = null;
     }
 
-
     public void refresh(Map<String, Object> configuration) {
         if (MapUtils.isEmpty(configuration)) {
             LOGGER.error("Received null or empty configuration during refresh.");
             return;
         }
 
-        Integer corbaTimeout = (Integer) configuration.get(CORBA_TIMEOUT);
-        if (corbaTimeout != null && corbaTimeout != this.corbaTimeout) {
-            setCorbaTimeout(corbaTimeout);
+        if (configuration.get(CORBA_TIMEOUT) != null) {
+            setCorbaTimeout((Integer) configuration.get(CORBA_TIMEOUT));
```

Comment:
We have historically seen issues with the map of <String, Object> since the only contract for the values is `Object` - I agree with glen to code defensively here